spring-pulsar/src/main/java/org/springframework/pulsar/annotation/PulsarListener.java

```diff
@@ -170,4 +171,12 @@
 	 */
 	String negativeAckRedeliveryBackoff() default "";
 
+	/**
+	 * The bean name or a 'SpEL' expression that resolves to a
+	 * {@link org.apache.pulsar.client.api.DeadLetterPolicy} to use on the consumer to
+	 * configure a maximum message redelivery.
```

Comment:
Close.... ;)

Reply:
Hehe. Sorry...
```
to use on the consumer to configure a maximum message redelivery..
```
As in "maximum message redelivery" is close to "dead letter policy"